### PR TITLE
Frenetic 3.4.0

### PR DESCRIPTION
**Pull request by arjunguha**

### CODE REVIEW

File: packages/frenetic/frenetic.3.4.0/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+build: [
+  ["./configure" "--%{pa_ounit:enable}%-tests" "--prefix" prefix ]
+  [make]
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "netkat"]
+]
+depends: [
+  "ocamlfind"
```
Comment (rgrinberg) [inline packages/frenetic/frenetic.3.4.0/opam]: Shouldn't this be a `{build}` dependency?

Comment (arjunguha) [inline packages/frenetic/frenetic.3.4.0/opam]: Thanks. I was not aware of that. Let me tweak.

Comment (arjunguha) [inline packages/frenetic/frenetic.3.4.0/opam]: Wait, I'm confused. Isn't everything a build-dependency?

Comment (rgrinberg) [inline packages/frenetic/frenetic.3.4.0/opam]: I think the difference is you wouldn't necessarily want frenetic to recompile if ocamlfind is updated.

Comment (arjunguha) [inline packages/frenetic/frenetic.3.4.0/opam]: Got it. thanks.

**Comment by dsheets** [comment]
FYI, even though it's marked as 4.01.0+, it will only work on 4.02.0+ because `openflow` requires 4.02.0+. Merging.

**Pull request merged**